apps/journeys-admin/src/components/ContainedIconButton/ContainedIconButton.tsx

```diff
@@ -0,0 +1,75 @@
+import { ReactElement, ReactNode } from 'react'
+import Box from '@mui/material/Box'
+import Typography from '@mui/material/Typography'
+import Card from '@mui/material/Card'
+import CardActionArea from '@mui/material/CardActionArea'
+import Stack from '@mui/material/Stack'
+import Icon from '@mui/material/Icon'
+import { ImageThumbnail } from '../ImageThumbnail'
+
+interface ContainedIconProps {
+  thumbnailIcon: ReactNode
+  actionIcon: ReactNode
```

Comment:
Lets make these optional, and fill it with some icons by default
```suggestion
  thumbnailIcon?: ReactNode
  actionIcon?: ReactNode
```

Reply:
fixed in 1dca4540ebe0a2853e4ff46a2f5f024b0153a5d0